test/ziggurat/streams_test.clj

```diff
@@ -24,6 +24,11 @@
                                     fix/silence-logging
                                     fix/mount-metrics]))
 
+(defn- start-mount
+  []
+  (mount/start-with-states [[#'ziggurat.messaging.consumer/consumers {:start (constantly nil)
```

Comment:
Replacing the rabbitmq consumers state with a dummy state, because consumers also require the queues to be present, without which they fail.